### PR TITLE
[Queues] fix omissions in get-started.md

### PR DESCRIPTION
**Pull request by marcduiker**
The guide now:
- Uses POST requests to trigger the producer Worker. 
- Uses the worker file structure and types that wrangler generates when `wrangler init` is used.

This fixes issue: #7482

### CODE REVIEW

File: content/queues/get-started.md [hunk 1]
```diff
@@ -146,7 +151,13 @@ Published <YOUR-WORKER-NAME> (0.29 sec)
   https://<YOUR-WORKER-NAME>.<YOUR-ACCOUNT>.workers.dev
 ```
 
-Copy your `*.workers.dev` subdomain and paste it into a new browser tab. Refresh the page a few times to start publishing requests to your queue. Your browser should return the `Success` response after writing the request to the queue each time.
+Now make a POST request to your worker using cURL:
```
Comment (deadlypants1973) [inline content/queues/get-started.md]:
```suggestion
Now make a POST request to your producer Worker using cURL:
```

File: content/queues/get-started.md [hunk 2]
```diff
@@ -192,7 +197,7 @@ To connect your queue to your consumer Worker, open your `wrangler.toml` file an
 
 ```toml
 [[queues.consumers]]
- queue = "<YOUR_QUEUE_NAME>"
+ queue = "YOUR_QUEUE_NAME"
```
Comment (deadlypants1973) [inline content/queues/get-started.md]:
```suggestion
 queue = "<YOUR_QUEUE_NAME>"
```

Adding <> back to indicate that this is a field the user needs to update.

**Review by deadlypants1973 — APPROVED** [review]

File: content/queues/get-started.md [hunk 3]
```diff
@@ -146,7 +151,13 @@ Published <YOUR-WORKER-NAME> (0.29 sec)
   https://<YOUR-WORKER-NAME>.<YOUR-ACCOUNT>.workers.dev
 ```
 
-Copy your `*.workers.dev` subdomain and paste it into a new browser tab. Refresh the page a few times to start publishing requests to your queue. Your browser should return the `Success` response after writing the request to the queue each time.
+Now make a POST request to your producer Worker using cURL:
```
Comment (deadlypants1973) [inline content/queues/get-started.md]:
```suggestion
After you have updated your `index.ts` file, make a POST request to your producer Worker using cURL:
```

**Review by deadlypants1973 — APPROVED** [review]

**Comment by deadlypants1973** [comment]
@elithrar to review

**Comment by elithrar** [comment]
Superseded by #7673 - which rewrote the examples to be easier to test in a browser (without a `POST` payload).

**Pull request closed without merging**